Contribute/Markdown-Conventions/index.md

```diff
@@ -38,7 +38,8 @@ or
 
 ## Internal links
 
-If you need to link between pages, always link relatively, and include the .md extension.
+If you need to link between pages, always link relatively, and include the .md extension. 
```

Comment:
The issue I raised was because of the text mentioning to include the .md extension, but because the extension in the code samples gets _lost in translation_ when this particular page is viewed on Our, it can create some confusion.  
I'd suggest we change it to something like _"always link relatively, optionally including the .md extension."_

(Also: There's a space missing between "required." and "Otherwise" in the second sentence :-)

Reply:
@greystate I have made the changes, please let me know if this looks good to be merged.